server/src/internalClusterTest/java/org/opensearch/explain/ExplainActionIT.java

```diff
@@ -187,21 +189,23 @@ public void testExplainWithSource() throws Exception {
             .get();
 
         refresh();
-        ExplainResponse response = client().prepareExplain(indexOrAlias(), "1")
-            .setQuery(QueryBuilders.matchAllQuery())
-            .setFetchSource("obj1.field1", null)
-            .get();
-        assertNotNull(response);
-        assertTrue(response.isMatch());
-        assertNotNull(response.getExplanation());
-        assertTrue(response.getExplanation().isMatch());
-        assertThat(response.getExplanation().getValue(), equalTo(1.0f));
-        assertThat(response.getGetResult().isExists(), equalTo(true));
-        assertThat(response.getGetResult().getId(), equalTo("1"));
-        assertThat(response.getGetResult().getSource().size(), equalTo(1));
-        assertThat(((Map<String, Object>) response.getGetResult().getSource().get("obj1")).get("field1").toString(), equalTo("value1"));
+        assertBusy(() -> {
+            ExplainResponse response = client().prepareExplain(indexOrAlias(), "1")
+                .setQuery(QueryBuilders.matchAllQuery())
+                .setFetchSource("obj1.field1", null)
+                .get();
+            assertNotNull(response);
+            assertTrue(response.isMatch());
+            assertNotNull(response.getExplanation());
+            assertTrue(response.getExplanation().isMatch());
+            assertThat(response.getExplanation().getValue(), equalTo(1.0f));
+            assertThat(response.getGetResult().isExists(), equalTo(true));
+            assertThat(response.getGetResult().getId(), equalTo("1"));
+            assertThat(response.getGetResult().getSource().size(), equalTo(1));
+            assertThat(((Map<String, Object>) response.getGetResult().getSource().get("obj1")).get("field1").toString(), equalTo("value1"));
+        });
 
-        response = client().prepareExplain(indexOrAlias(), "1")
+        ExplainResponse response = client().prepareExplain(indexOrAlias(), "1")
```

Comment:
why was this change needed? response in line 68 has already been reused once on line 87.